Controllers/AuthController.cs

```diff
@@ -28,27 +28,30 @@ public AuthController(ICustomersService service, IConfiguration config)
         public ActionResult Login([FromBody] CustomerLogin body)
         {
             var currentUser = _service.Authenticate(body);
+
             if(currentUser == null) 
             {
-                return Unauthorized("Invalid username or password");
+                var unauthorizedResponseJson = new { message = "Invalid username or password" };
+                return Unauthorized(unauthorizedResponseJson);
             }
 
-            return Ok(GenerateToken(currentUser));
+            var responseJson = new { token = GenerateToken(currentUser) };
```

Comment:
same for this, I think I remember hearing in lectures that controllers should only call a service and the main job of this class is defining input, output, routes, etc...

Reply:
sounds good.